frontend/web/components/modals/CreateRole.tsx

```diff
@@ -373,6 +371,44 @@ const CreateRole: FC<CreateRoleType> = ({
           id='description'
           placeholder='E.g. Some role description'
         />
+        <FormGroup className='mb-5 setting'>
+          <InputGroup
+            title={'Tags'}
+            tooltip={'Select the project where you want to manage your tags'}
```

Comment:
This looks wrong, we need to provide good context here about what adding tags to a role means. Can we link out to some documentation?

Reply:
I added documentation